zipkin-lens/src/components/App/index.js

```diff
@@ -4,8 +4,9 @@ import { BrowserRouter, Route } from 'react-router-dom';
 
 import Layout from './Layout';
 import BrowserContainer from '../../containers/Browser/BrowserContainer';
-import DetailedTraceSummaryContainer from '../../containers/DetailedTraceSummary/DetailedTraceSummaryContainer';
+import TracePageContainer from '../../containers/TracePage/TracePageContainer';
```

Comment:
I refactored to reuse `DetailedTraceSummary`.

* Remove a process of fetching trace data from `DetailedTraceSummary` component.
* Make a new component `TracePage`, and add that process to the component.